pyrobuf/parse_proto.py

```diff
@@ -521,14 +566,21 @@ def _parse_enum(self, current, tokens, scope, current_message=None):
         token = next(tokens)
         assert token.token_type == 'LBRACE', "missing opening brace on line {}: '{}'".format(
             token.line + 1, self.lines[token.line])
+        previous = self.LBrace(-1) 
+        setDefault = False
+
+        for token in tokens:
+            if self._handleComment(token,previous):
+                previous = token
+                continue
 
-        for num, token in enumerate(tokens):
             if token.token_type == 'ENUM_FIELD':
-                if num == 0:
+                if (setDefault is False):
```

Comment:
Can you explain the use of `setDefault` here? It's not clear to me. Also would prefer `if not setDefault` without the parens.

Reply:
I only see, that you enumerate to get a number for every token, but you access/check only the first `num == 0` and set default. For this simple task, a status variable `set_default` is more understandable. On the other side, with comment tokens, the first token in Enum-Definition can be a comment, which gets the number one.

enum Level
{
   // any comment, the first token in parse enum
  Normal = 0; // first enum, but second token  in parse enum
  Super   = 1;
}

Reply:
Okay.